### PR TITLE
S01 survey

### PR DESCRIPTION
**Pull request by khaliev**
questions and styles addes

### CODE REVIEW

File: frontend/src/components/Navbar.css [hunk 1]
```diff
@@ -1,35 +1,39 @@
+* {
+  margin: 0;
+  padding: 0;
+}
 navbar {
-  height: 100rem;
-  width: 10rem;
+  height: 10%;
```
Comment (rocambille) [inline frontend/src/components/Navbar.css:6]: remove fixed height

File: frontend/src/components/Navbar.css [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
   align-items: center;
+  justify-content: space-between;
 }
 
 .liste {
   display: flex;
-  height: 35rem;
-  flex-direction: column;
-  justify-content: space-between;
+  height: 10%;
```
Comment (rocambille) [inline frontend/src/components/Navbar.css:17]: same here

File: frontend/src/components/Obligatory.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+  const handleNextSection = () => {
+    if (question1 && question2 && question3 && question4) {
+      setObligatoryCompleted(true);
+    } else {
+      alert("Veuillez répondre à toutes les questions obligatoires");
+    }
+  };
+
+  return (
```
Comment (rocambille) [inline frontend/src/components/Obligatory.jsx:36]: you should not have an Obligatory, Essential and an Optional component, but a Question component with a `type` prop

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by Justine-Merlin** [comment]
Still conflicts to resolve 👀

**Pull request closed without merging**